src/Security/SecurityConfigUpdater.php

```diff
@@ -119,6 +140,31 @@ public function updateForAuthenticator(string $yamlSource, string $firewallName,
         return $this->manipulator->getContents();
     }
 
+    public function updateForLogout(string $yamlSource, string $firewallName): string
+    {
+        $this->manipulator = new YamlSourceManipulator($yamlSource);
+
+        if (null !== $this->ysmLogger) {
+            $this->manipulator->setLogger($this->ysmLogger);
+        }
+
+        $this->normalizeSecurityYamlFile();
+
+        $newData = $this->manipulator->getData();
+
+        $newData['security']['firewalls'][$firewallName]['logout'] = ['path' => 'app_logout'];
+        $newData['security']['firewalls'][$firewallName]['logout'][] = $this->manipulator->createCommentLine(
+            ' where to redirect after logout'
+        );
+        $newData['security']['firewalls'][$firewallName]['logout'][] = $this->manipulator->createCommentLine(
+            ' target: app_any_route'
+        );
```

Comment:
This is duplicated above. I know we're going to deprecate that code that uses the code above, but probably we should still isolate into a private method and call this from both places to avoid duplication, even temporary.

Reply:
done